packages/@ember/octane-app-blueprint/files/src/resolver.js

```diff
@@ -3,26 +3,15 @@ import buildResolverConfig from 'ember-resolver/ember-config';
 import config from '../config/environment';
 
 let moduleConfig = buildResolverConfig(config.modulePrefix);
+
 /*
  * If your application has custom types and collections, modify moduleConfig here
- * to add support for them.
+ * to add support for them:
+ *
+ *  moduleConfig.types = Object.assign(moduleConfig.types, {
+ *    'validator': { definitiveCollection: 'validators' },
```

Comment:
Validator?
Who dis?

Reply:
Maybe if this is meant to serve as an example, we should link to the resolver's config?

Reply:
I thought about that too, but I think we should also provide an example to show how it can be configured.

I could add more examples or add the link to the resolver too, but I thought that the resolver location may change.

Reply:
That's a fair point. Maybe a few additional examples, but commented out?

Reply:
oh this is a comment. lol. cool